src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -346,7 +353,8 @@
      * Gets the villager {@link Profession} with the specified id.
      *
      * @param id The id of the profession to return
-     * @return The profession with the given id or Optional.absent() if not found
+     * @return The profession with the given id or Optional.absent() if not
+     *         found
```

Comment:
Yeah... It's just that when I save GameRegistry, my formatter wants to fix all the line lengths... I'm looking for a good solution.